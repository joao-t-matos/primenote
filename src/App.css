.menu {
  position: absolute;
  overflow: hidden;
  top: 0;
  left: 0;
  width: 0;
  height: 100vh;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  z-index: 99;
  transition: all .5s ease-in-out;
}


.menu-dark {
  color: rgba(37, 37, 37, 0.98) !important;
  background-color: rgb(228, 228, 228, 0.98);
}

.menu-light {
  color: rgb(228, 228, 228, 0.98) !important;
  background-color: rgb(37, 37, 37, 0.98);
}

.menu-row {
  height: 10%;
  margin: 10px 0;
}

button.menu-row {
  font-size: 40px;
  color: inherit;
  background-color: transparent;
  border: none;
  transition: color .3s;
}

button.menu-row:hover {
  color: #9c27b0;
}

.menu-button-container {
  color: inherit;
  width: 100%;
  height: auto;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: flex-end;
}

.menu-close{
  font-size: 30px;
  color: inherit;
  border: none;
  background-color: inherit;
}

.menu-button {
  position: absolute;
  top: 10px;
  left:10px;
  width: 50px;
  height: 50px;
  border-radius: 50%;
  cursor: pointer;
}

.dark-mode-switch{
  display: flex;
  align-items: center;
}

.button-dark {
  color: rgb(228, 228, 228);
  background-color: rgb(37, 37, 37);
  border: 2px solid rgb(228, 228, 228);
}

.button-light {
  color: rgb(37, 37, 37);
  background-color: rgb(228, 228, 228);
  border: 2px solid rgb(37, 37, 37);
}

.markdown-area {
  width: 80%;
  height: 80%;
  display: flex;
  border-radius: 20px;
  transition: all .5s;
}

.markdown-area-light {
  background-color: rgb(37, 37, 37);
}

.markdown-area-dark {
  background-color: rgb(228, 228, 228);
}

.markdown-area textarea {
  resize: none;
  background-color: rgb(209, 209, 209);
  border: 10px solid rgb(209, 209, 209);
  border-radius: 20px;
}

.input-area  {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 50%;
  height: 100%;
}

.input-area textarea {
  width: 90%;
  height: 90%;
  color: rgb(84, 84, 84);
  transition: all .5s;
}

.input-area textarea:focus-visible {
  outline: none;
  background-color: rgb(202, 186, 203);
  border-color: rgb(202, 186, 203);
}

.result-area{
  width: 50%;
  display: flex;
  align-items:center;
  flex-direction: column;
}

.result-area textarea{
  width: 90%;
  height: 40%;
  margin: auto;
}

@keyframes menu-appear {
  from {opacity: 0%;}
  to {opacity: 100%;}
}