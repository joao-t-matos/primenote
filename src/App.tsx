import React, {FC, useState} from 'react';
import './App.css';
import Home from './pages/Home';

enum navigationScreens {
  HOME = "HOME",
  ABOUT = "ABOUT"
}

const App: FC = () => {
  const [navigation, setNavigation] = useState<String>();
  return (
    <div className="App">
      <Home/>
    </div>
  );
}

export default App;
