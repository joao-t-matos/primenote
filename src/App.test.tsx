import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

test('renders Home as the first screen', () => {
  render(<App />);
  const linkElement = screen.getByText(/Home/i);
  expect(linkElement).toMatchSnapshot('Home');
});
