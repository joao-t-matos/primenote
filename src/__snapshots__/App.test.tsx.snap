// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home as the first screen: Home 1`] = `
<h1
  class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
>
  Home
</h1>
`;
